runtime/capabilitycontrollers_test.go

```diff
@@ -208,200 +215,436 @@ func TestRuntimeCapabilityControllers(t *testing.T) {
 				require.NoError(t, err)
 			})
 
-			t.Run("get and check existing, with valid type", func(t *testing.T) {
+			t.Run("get linked capability", func(t *testing.T) {
 
 				t.Parallel()
 
-				err, _, _ := test(
-					fmt.Sprintf(
-						// language=cadence
-						`
-                          import Test from 0x1
+				t.Run("storage link", func(t *testing.T) {
+					err, _, _ := test(
+						fmt.Sprintf(
+							// language=cadence
+							`
+                              import Test from 0x1
+
+                              transaction {
+                                  prepare(signer: AuthAccount) {
+                                      let storagePath = /storage/r
+                                      let publicPath = /public/r
+                                      let resourceID = 42
+
+                                      // Arrange
+                                      Test.createAndSaveR(id: resourceID, storagePath: storagePath)
+                                      signer.link<&Test.R>(publicPath, target: storagePath)!
+
+                                      // Act
+                                      let gotCap1: Capability<&Test.R>? =
+                                          %[1]s.getCapability<&Test.R>(publicPath)
+                                      let gotCap2: Capability<&Test.R>? =
```

Comment:
Is there a test that does the opposite of this? i.e: publishing a capability through the new API, and trying to get it from the old `getCapability()` API

Reply:
I'm not quite sure in which PR such a test is added, but the branch of the final PR has such a test case (if I'm understanding correctly): https://github.com/onflow/cadence/blob/bastian/cap-cons-11/runtime/capabilitycontrollers_test.go#L266-L344

(`getCapability` is basically just a constructor for `Capability`, it does not interact with storage at all, so calling it before or after `issue` is the same).